test/step.sh

```diff
@@ -0,0 +1,48 @@
+#!/usr/bin/env bash
```

Comment:
This script seems to just call `${ROOT}/test/$@.sh`. Any reason for it being an explicit script instead of callers just invoking `./test/step.sh <stepname>`?